src/wfs/src/main/java/org/geoserver/wfs/CapabilitiesTransformer.java

```diff
@@ -1392,8 +1395,9 @@ protected void internationalServiceProvider(ContactInfo contact) {
                                 contact.getInternationalAddressCountry()));
                 element(
                         "ows:ElectronicMailAddress",
-                        internationalContentHelper.getNullableString(
-                                contact.getInternationalContactEmail()));
+                        StringEscapeUtils.escapeXml10(
```

Comment:
The contact mail is used in other capabilities document generators as well (WCS, WMS) but wondering if this escaping is actually useful, I think the low-level XML encoder here should already be doing the necessary escaping for you. Do you have a way to actually do harm using the XML documents?

Reply:
Good point ... I am unaware of a way to attack using XML document? Do you suggest rolling back this change?!

Reply:
If it's not needed, better to back it out, yes (if you find a way to move an attack using XML documents instead, it needs tests and be replicated in all other places a caps document emits the mail address).